### ISSUE TITLE
Debugger should allow variable creation in the REPL window

### ISSUE THREAD

**Issue by gregg-miskelly** [OmniSharp/omnisharp-vscode#3503]
## Environment data
C# Extension version: 1.21.9

## Steps to reproduce
* Create the following project.
* Set a breakpoint at the specified line
* Evaluate `var x = bytes.Length();`

```
using System;
using System.Buffers;
using System.Text;

namespace DNC31Con
{
    class Program
    {
        static void Main(string[] args)
        {
            byte[] bytes = { (Byte)'h', (Byte)'e', (Byte)'l', (Byte)'l', (Byte)'o', (Byte)'!' };
            ReadOnlySequence<Byte> sequence = new ReadOnlySequence<byte>(bytes);

            Console.WriteLine("Hello World!"); // breakpoint here
        }
    }
}
```

## Expected  behavior
Implicit variable 'x' should be created.

## Actual behavior
Evaluation fails

**Comment by cartermp** [comment]
@gregg-miskelly @WardenGnaw has this gone in, given the label re: fixed pending insertion?

**Comment by gregg-miskelly** [comment]
This went in sometime while I was on leave, so I am not sure what the right release was, but it should be fixed in 1.21.18.
